cylc/flow/parsec/fileparse.py

```diff
@@ -471,6 +471,11 @@ def read_and_proc(
     template_vars = merge_template_vars(template_vars, extra_vars)
     template_vars['CYLC_TEMPLATE_VARS'] = template_vars
 
+    # Add CYLC_SOURCE_DIRECTORY to template variables.
+    sourcepath = get_workflow_source_dir(Path(fpath).parent)[1]
+    template_vars['CYLC_SOURCE_DIRECTORY'] = (
```

Comment:
There's a convention to prefix workflow variables with `CYLC_WORKFLOW` which would be worth upholding for consistency.

We have a variable called `CYLC_WORKFLOW_RUN_DIR`, perhaps this should be `CYLC_WORKFLOW_SRC_DIR` to match?